### DISCUSSION TITLE
creating a page

### DISCUSSION THREAD

**Discussion by ghost** [community/community#58176] [New to GitHub] [answered]
Hello👋, I have just created a github account with 3 repositories! However I made the mistake of my first repository name does not match my username 😔. But I had created a second repository with the same name as my username, so I just used this one isntead. I can now see that it says a page is deployed, so I assume what I did worked?  But if I type on my browser https://valeridev.github.io  it comes back as 404 error. Howerver if I type on my browser https://valeridev.github.io/valeridev it works. 
I appreciate everyone's suggestion and effort to help me with that. kindest regards! Valeri

**Comment by Gustavonn07** [comment]
Oi tudo bem? (respondi em portugues, pois vi que você sabe falar portugues) Quando você cria um repositorio com o seu nome ele é um repositorio especial que serve como um portifolio no github, no seu caso o seu repositorio mostra isso certo?
<hr>

![image](https://github.com/community/community/assets/84361085/54e2f764-ef51-440c-9762-47e81726cf6e)
<hr>

Você pode nesse repositorio alterar como quiser e deixar estilizado (vou usar exemplo do meu)
<hr>

![image](https://github.com/community/community/assets/84361085/79ee8f0e-f64d-4c2c-a191-c1e0ca146d6f)

<hr>

Nesse tipo de repositório você não precisa se preocupar com sites, ja que todos os usuários do github podem ver do jeito que eu vi!
Caso você queira criar uma pagina web com o github certifique-se que o seu html principal se chama index.html e siga os seguintes passos:

1. Entre no repositório escolhido e siga a setinha até configurações:
<hr>

![image](https://github.com/community/community/assets/84361085/6e51055e-ce1a-4eb5-92d9-4834d40d4377)

<hr>

2. Siga a setinha até pages:

<hr>

![image](https://github.com/community/community/assets/84361085/01bbf08b-0a53-417e-99e6-a8834de97471)

<hr>

3. Em pages deixe da mesma forma que eu fiz:

<hr>

![image](https://github.com/community/community/assets/84361085/7fd63db2-d15f-47ab-95c6-df6f40a6a6d3)


<hr>

4.  As vezes demora de 3 a 10min para criar uma página, mas quando criar aparece o link da sua nova página na web (Lembrando não saia da tela do github pages se não buga)

Espero ter ajudado! Caso eu não respondi certo a pergunta favor me avisar.

**Comment by ghost** [comment]
Oiii! Tudo bem! Obrigada. Valeu pela aula. Então o html principal com index.html é o nome do repositorio valeridev? Porem eu não tenho nenhum código nele, somente um readme.md. Eu teria que por um codigo de um projeto la no repositório valeridev chamado index.html? desculpa a confusao!

**Reply by Gustavonn07** [reply]
Nesse caso, o README (um documento md) ele serve como documentação / apresentação do seu repositório por isso ele aparece para todos assim que você entra em um repositório, como o repositório com o seu nome é um especial esse README.md aparece na página inicial do seu perfil (além da página do repositorio).

Tirando umas duvidas:
1. README é diferente de index.html o HTML é um arquivo de Hyper Text Markup Language
 que serve para criação de sites junto com o CSS e JAVASCRIPT

2. README é um arquivo md que contém informação sobre outros arquivos num diretório ou arquivo e é comummente distribuído com softwares de computador

Caso possua mais alguma dúvida pode perguntar sem problemas!

**Comment by ghost** [comment]
👋Então eu tenho que criar um file no repositorio valeridev com o nome
index.html ?

**Reply by Gustavonn07** [reply, marked as answer]
No caso como esse repositório possui o seu nome (Um repositório especial que aparece no seu perfil principal) não é necessário adicionar html, apenas o README (O que ja foi adicionado).

Caso você queira criar um repositório com o seu nome e fazer ele virar um site eu recomendo usar um nome um pouco diferente do seu perfil (em vez de usar valeriedev use valeriedev-reposiositorio)

Então, não se preocupe em adicionar nada (Como o index.html, style.css ou script.js,...) nesse repositorio com seu nome, apenas se preocupe em editar o README para adicionar estilização (Caso vc adicione algo seria mais um arquivo para APIs)

Eu recomendo fortemente essa youtuber (Rafaella Balerini) para ensinar mais sobre edição de arquivos readme: https://www.youtube.com/watch?v=TsaLQAetPLU 

Qualquer coisa so falar!

**Comment by ghost** [comment]
Obrigada! Fiz umas modificaçōes lá com sua ajuda! happy coding!👋

**Reply by ghost** [reply]
agora q vi o link do youtube, vou visitar e ir melhorando o q ja esta feito. obrigada.

**Reply by Gustavonn07** [reply]
De nadaa, caso tenha dado tudo certo você poderia marcar meu comentário como respondido para outras pessoas já saberem que deu certo! (e tambem iria me ajudar bastante 😅)

Boa sorte nos seus códigos 😄!!
